openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -58,6 +59,9 @@
     @CommandLine.Option(names = {"-i", "--input"}, description = "Generating the client and service both files")
     private boolean inputPath;
 
+    @CommandLine.Option(names = {"-l", "--license"}, description = "Location of the license header file")
+    private boolean licensePath;
```

Comment:
This can be directly assigned to the `String` type. Is there any reason to set this as a `boolean` value? :)

Reply:
Yes, that's true. Updated